src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -1675,7 +1675,14 @@ void ReportCoilSelection::setCoilHeatingCapacity(
             }
         } else if (state.dataSize->ZoneEqFanCoil) {
             if (c->coilDesEntTemp == -999.0) { // don't overwrite if already set directly by setCoilEntAirTemp
-                Real64 desOAFlowFrac = state.dataSize->FinalZoneSizing(curZoneEqNum).DesHeatOAFlowFrac;
+                Real64 desOAFlowFrac = 0.0;
```

Comment:
Much opportunity to use `auto &` shortcuts in this module there is.

Reply:
I agree. I will get back to it when the functional changes are complete.